content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,94 @@
+
+### External Istiod promoted to Beta
+
+As with VMs, management of external clusters is vital as Mesh Admins increasingly need to expand their service meshes
+to include remote clusters for many applications (e.g. global distribution, high availability). By promoting this
+feature to Beta we recognize this as a must for adoption and to promote widespread-usage.
+
+## service-apis promoted to Alpha
+
+Support for [Kubernetes service-apis](https://kubernetes-sigs.github.io/service-apis/) has been an active area of
+development since Istio 1.8. These are marketed as the next generation of the Ingress API, and draw inspiration from
+Istio's Gateway and VirtualService. The Istio community is eager to evolve with Kubernetes and these CRDs in upcoming
+releases.
+
+## Third-party providers in Authorization Policy
+
+Authorization Policy has been changed to allow
+[third-party providers](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-ExtensionProvider)
+using [CUSTOM](/docs/reference/config/security/authorization-policy/#AuthorizationPolicy-Action) actions. This allows
+Mesh Admins to use external providers to check if an incoming HTTP request is authorized. This provider must
+implement the [Envoy ext_authz filter](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter)
+authorization check either using HTTP or gRPC.
+
+### Telemetry v2 Request Classification promoted to Beta
+
+Telemetry v2 will see some major improvements in 2021. To jumpstart the year,
+[Request Classification](/docs/tasks/observability/metrics/classify-metrics/) has been promoted to Beta. Promotion of
+this feature will enable users to better monitor the health and well-being of their service mesh.
+
+# Complete support for out-of-mesh metadata for outgoing traffic
+
+?
+
+## Mirroring of images on gcr.io
+
+Due to Docker Hub's rate-limiting policy the Istio community now also hosts publishes images on `gcr.io/istio-release`.
+If you are experiencing issues with failed downloads in your environment, please set the hub accrordingly. Similar to
+`istioctl install --set hub=gcr.io/testing`.
+
+## istioctl updates
+
+Istioctl continues to make major strides in aiding users to be a one-stop tool for users. Key features include
+the addition of the `verify-install` command that notify users of configuration errors; analyze now has the
+ability to deprecated or alpha-level annotations are present; and a bevy of other commands to help with
+multicluster environments, debugging and installation.
+
+## Join the Istio community
+
+Our [Community Meeting](https://github.com/istio/community#community-meeting) happens on the fourth Thursday of the month, at 10 AM Pacific. Due to US Thanksgiving, we've moved this month's meeting forward one week to the 19th of November. If you can't make it, why not join the conversation at [Discuss Istio](https://discuss.istio.io/), or join our [Slack workspace](https://slack.istio.io/)?
```

Comment:
thanksgiving mention seems wrong?

Reply:
Bad copy/paste. I added info related to IstioCon 2021.